configs/evonet/drive/tendermint/config.toml

```diff
@@ -50,17 +50,17 @@ log_format = "plain"
 genesis_file = "config/genesis.json"
 
 # Path to the JSON file containing the private key to use as a validator in the consensus protocol
-priv_validator_key_file = "/data/priv_validator_key.json"
+priv_validator_key_file = "/tendermint/priv_validator_key.json"
```

Comment:
What if we just make all paths as in default tendermint config and mount `$HOME/.tendermint` to volume?

```
# NOTE: Any path below can be absolute (e.g. "/var/myawesomeapp/data") or
# relative to the home directory (e.g. "data"). The home directory is
# "$HOME/.tendermint" by default, but could be changed via $TMHOME env variable
# or --home cmd flag.
```

Reply:
- Mounting the volume at `$HOME/.tendermint` doesn't work because upstream [defines](https://github.com/tendermint/tendermint/blob/master/DOCKER/Dockerfile#L9) `ENV TMHOME /tendermint` so we should mount the docker volume here to store all mutable state.
- Inside this, it works better to bind mount the dir instead of individual files to avoid implicitly creating the `config` dir as root inside the container. Instead, it will take the uid:gid of the host system. We shouldn't write anything to this dir or it will appear at the path `configs/${NETWORK:?err}/drive/tendermint`.
- For this reason, not all paths can be set to default, because 3 files (`priv_validator_key.json`, `node_key.json`, `addrbook.json`) are generated on init and therefore mutable. They should live in the `data` dir so they are cleaned up when the volume is removed.

If we bind mount the `config.toml` and `genesis.json` files then `config` owner is root and the 3 files cannot be written, resulting in container crash.